lib/AST/PluginRegistry.cpp

```diff
@@ -214,7 +214,7 @@ LoadedExecutablePlugin::PluginProcess::~PluginProcess() {
   close(input);
   close(output);
 #endif
-  llvm::sys::Wait(process, /*SecondsToWait=*/0);
+  llvm::sys::Wait(process, /*SecondsToWait=*/1);
```

Comment:
Worth adding a comment similar to the PR description here?

Reply:
Good idea. Done 👍